### ISSUE TITLE
Travis build fails when installing packages

### ISSUE THREAD

**Issue by wleoncio** [SystemsBiologist/CellMigRation#13]
This issue was already raised when discussing a different, closed issue (#8), so here's a separate submission for organizational purposes.

Long story short, CellMigRation is failiing on TravisCI. This is the relevant part of the log (full log [here](https://travis-ci.org/github/ocbe-uio/CellMigRation/jobs/664794192?utm_medium=notification&utm_source=github_status)):

```
Error: package or namespace load failed for ‘sm’:
 .onLoad failed in loadNamespace() for 'tcltk', details:
  call: NULL
  error: (converted from warning) no DISPLAY variable so Tk is not available
Error: package ‘sm’ could not be loaded
Execution halted
ERROR: lazy loading failed for package ‘vioplot’
* removing ‘/home/travis/R/Library/vioplot’
Error in i.p(...) : 
  (converted from warning) installation of package ‘vioplot’ had non-zero exit status
Calls: <Anonymous> ... with_rprofile_user -> with_envvar -> force -> force -> i.p
Execution halted
The command "Rscript -e 'deps <- remotes::dev_package_deps(dependencies = NA);remotes::install_deps(dependencies = TRUE);if (!all(deps$package %in% installed.packages())) { message("missing: ", paste(setdiff(deps$package, installed.packages()), collapse=", ")); q(status = 1, save = "no")}'" failed and exited with 1 during .

Your build has been stopped.
```

This can possibly be fixed by customizing the [.travis.yml](https://github.com/ocbe-uio/CellMigRation/blob/master/.travis.yml) file.

I'll assign myself and work on this, but this takes lower priority than the work I'm doing on the [shiny branch](https://github.com/ocbe-uio/CellMigRation/tree/shiny). Any help is welcome, please use the [travis_issue_13 branch](https://github.com/ocbe-uio/CellMigRation/tree/travis_issue_13) to commit changes related to this issue.

P.S.: it's possible other errors are triggered after these ones are fixed. Hopefully they are not so different or hard and the whole "travis build" problem can be addressed in this Issue.

**Comment by wleoncio** [comment]
Made some progress by adding the `xvfb` (X11 Virtual Framebuffer) service. Source: https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-xvfb-to-run-tests-that-require-a-gui

**Comment by wleoncio** [comment]
Fixed on d92c26e41c311fb9e1b4ba0aca8e5cd208d798d1.
